### PR TITLE
Add Nathan's changes

### PR DESCRIPTION
**Pull request by funkna**
Hey guys, quick recap on what I did here:
- Added data page 1 message handling in the WHIM Channel class.
- Added ANT message-based transitions between different views.
- Updated some of the view layouts.
- I changed the App list logo.

### CODE REVIEW

**Comment by parknick** [comment]
Looks good after first pass! I'll pull down these commits and test em out tomorrow - may have a few comments after that, but this matches what I had in mind.

File: source/WHIMChannel.mc [hunk 1]
```diff
@@ -73,9 +73,25 @@ class WHIMChannel extends Ant.GenericChannel
...
+				        var view = new SensorDetailsView();
+				        WatchUi.switchToView(view, new SensorDetailsDelegate(view), WatchUi.SLIDE_IMMEDIATE);
+	                }
+	                
+	                if (impacts != payload[1]) {
+	                	System.println("New Data!");
+			        	impacts = payload[1];
+			        	WatchUi.requestUpdate();
+			        }
+			    }
```
Comment (parknick) [inline source/WHIMChannel.mc]: Add an else case, which prints that an undefined/unrecognized data page has been received

File: source/SensorDetailsView.mc [hunk 2]
```diff
@@ -1,38 +1,50 @@
...
+    	if (impacts > 0) {
+    		View.findDrawableById("status").setColor(Graphics.COLOR_RED);
+    		View.findDrawableById("status").setText(Rez.Strings.text_status_urgent);
+    	}
+    	else {
+    		View.findDrawableById("status").setColor(Graphics.COLOR_GREEN);
+    		View.findDrawableById("status").setText(Rez.Strings.text_status_safe);
+    	}
+    	
+
```
Comment (parknick) [inline source/SensorDetailsView.mc]: nit: Delete one of these newlines

File: source/WHIMChannel.mc [hunk 3]
```diff
@@ -89,11 +105,13 @@ class WHIMChannel extends Ant.GenericChannel
                             System.println( "EVENT_TX" );
                             break;
                         case Ant.MSG_CODE_EVENT_CHANNEL_CLOSED:
-                            System.println( "CHANNEL_CLOSED" );
-                            checkChannelClosure();
+                            System.println( "TIMER_EXPIRED_REOPEN_CHANNEL" );
+                            open();
```
Comment (parknick) [inline source/WHIMChannel.mc:109]: This is okay for now, but we won't always want to reopen when the channel closes. I have an idea for how we'll want to do this, so it may be best for me to make a separate commit for that (this commit is already pretty big anyway!).
We can talk more about this tomorrow though, along with everything else.

**Review by parknick — CHANGES_REQUESTED** [review]
Left a couple minor comments that should be addressed before this gets submitted, and one just FYI.

In tomorrow's meeting, our team should talk about tabs vs. spaces in our code, as well as commit message format. Right now there's a mix of tabs and spaces in some files (see screencap below for example), which I think is why some indentations aren't lining up here on GitHub. So we should wait for that discussion before merging this too!

image

File: source/WHIMChannel.mc (hunk 1 above)
Comment (funkna) [reply source/WHIMChannel.mc]: Now it is done.

File: source/SensorDetailsView.mc (hunk 2 above)
Comment (funkna) [reply source/SensorDetailsView.mc]: Done

File: source/WHIMChannel.mc (hunk 3 above)
Comment (funkna) [reply source/WHIMChannel.mc:109]: Acknowledged.

We will add the solution we discussed in a future commit.

File: source/WHIMChannel.mc [hunk 4]
```diff
@@ -54,31 +54,47 @@ class WHIMChannel extends Ant.GenericChannel
...
+
+                    if (impacts != payload[1]) {
+                        System.println("New Data!");
+                        impacts = payload[1];
+                        WatchUi.requestUpdate();
+                    }
+                }
+                else {
+                    System.println("Unrecognized data page received.");
+                }
```
Comment (parknick) [inline source/WHIMChannel.mc:94]: Nice! 👍

**Review by parknick — APPROVED** [review]
